csrc/scheduler/matmul_heuristic.h

```diff
@@ -64,16 +64,18 @@ class MatmulParams : public HeuristicParams {
     }
   };
 
-  //! This is the maximum vectorization supported by the inputs.
+  //! This is the maximum vectorization supported by the inputs. This refers to
+  //! the number of data elements loaded simultaneously, not the number of
+  //! bytes.
```

Comment:
Could you add a comment describing why the default value is 16? This can be confusing.

Reply:
I will add that comment; I don't think there's a really compelling reason to change it. I agree that it's confusing, and I've had to set it manually to fix a lot of the benchmarks and tests, but now that it's set I think it might be nice because it is a bit more explicit. With automatic scheduling it should always be set to a sane value. One reason I set it higher to a value that I knew was invalid was as a signal that we've set it properly in each of those cases.